Assets/Scripts/StandardSamples/UI/Storage/Player/UIPlayerDataStorageMenu.cs

```diff
@@ -89,12 +87,12 @@ private void Start()
         protected override void OnDestroy()
         {
             base.OnDestroy();
-            EOSManager.Instance.RemoveManager<EOSPlayerDataStorageManager>();
+            EOSManager.Instance.RemoveManager<PlayerDataStorageService>();
```

Comment:
Can this be removed now that we're using Singleton?

Reply:
Yep, done.